app/src/main/res/values-de/strings.xml

```diff
@@ -1,78 +1,217 @@
-<?xml version='1.0' encoding='utf-8'?>
```

Comment:
I notice only 4 files have changed: why aren't the remaining locales affected? Specifically:
- ja
- es-rES
- it